src/Smart.Design.Razor/TagHelpers/Base/BaseSmartFormGroupTagHelper.cs

```diff
@@ -0,0 +1,69 @@
+using Microsoft.AspNetCore.Mvc.Rendering;
+using Microsoft.AspNetCore.Mvc.TagHelpers;
+using Microsoft.AspNetCore.Mvc.ViewFeatures;
+using Microsoft.AspNetCore.Razor.TagHelpers;
+using Smart.Design.Razor.TagHelpers.Extensions;
+using Smart.Design.Razor.TagHelpers.Html;
+
+namespace Smart.Design.Razor.TagHelpers.Base
+{
+    /// <summary>
+    /// Abstract base class <see cref="ITagHelper" /> to render smart design form groups with asp-for, helper-text , label, value, value attributes.
```

Comment:
Here, please capitalize the name of the company, Smart.

Reply:
Yeah you right dunno what I didnt do that,